streaming/streaming.go

```diff
@@ -447,20 +452,62 @@ func getStreamingConfig(sourceProfile profiles.SourceProfile, targetProfile prof
 	}
 }
 
+func CreateStreamingConfig(pl profiles.DataShard) StreamingCfg {
+	//create dataflowcfg from pl receiver object
+	dataflowCfg := DataflowCfg{}
+	inputDataflowConfig := pl.DataflowConfig
+	dataflowCfg.Location = inputDataflowConfig.Location
+	dataflowCfg.Network = inputDataflowConfig.Network
+	dataflowCfg.HostProjectId = inputDataflowConfig.HostProjectId
+	dataflowCfg.Subnetwork = inputDataflowConfig.Subnetwork
+	//create src and dst datastream from pl receiver object
+	datastreamCfg := DatastreamCfg{}
+	//set stream config
+	datastreamCfg.StreamLocation = pl.StreamLocation
+	//set src connection profile
+	inputSrcConnProfile := pl.SrcConnectionProfile
+	srcConnCfg := SrcConnCfg{}
+	srcConnCfg.Location = inputSrcConnProfile.Location
```

Comment:
nit: Might be more concise to be put the parameter as a part of the creation of the struct
Eg:
SrcConnCfg{Location: "asd", Name: "pqr}

Reply:
Hehe, brain fart 😛
Thanks for highlighting! Updated!